models/deepseek/v4/moe_ep.py

```
@@ -0,0 +1,1144 @@
+    sw2_bf16 = (torch.randn(D, MOE_INTER) / MOE_INTER ** 0.5).to(torch.bfloat16)
+    sw1_i8, sw1_s = _quant_w_per_channel(sw1_bf16)
+    sw3_i8, sw3_s = _quant_w_per_channel(sw3_bf16)
+    sw2_i8, sw2_s = _quant_w_per_channel(sw2_bf16)
+    sw1_i8 = sw1_i8.unsqueeze(0).expand(N_RANKS, -1, -1).contiguous()
+    sw1_s = sw1_s.unsqueeze(0).expand(N_RANKS, -1).contiguous()
+    sw3_i8 = sw3_i8.unsqueeze(0).expand(N_RANKS, -1, -1).contiguous()
+    sw3_s = sw3_s.unsqueeze(0).expand(N_RANKS, -1).contiguous()
+    sw2_i8 = sw2_i8.unsqueeze(0).expand(N_RANKS, -1, -1).contiguous()
+    sw2_s = sw2_s.unsqueeze(0).expand(N_RANKS, -1).contiguous()
+
+    out = lambda name, shape, dtype: TensorSpec(name, shape, dtype, is_output=True)
```

Ruff (0.15.14)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `out` as a `def`

(E731)